readme.md

```diff
@@ -59,7 +59,13 @@ PRs and other contributions are very welcome!
 
 ### Building the project
 
-Run `npm run build` from the root directory.
+Run these instructions from the root directory (where sample.cadl file is a CADL file you want to emit to RDF):
+```
+module load ossjs/node/16.13.2

```

Comment:
Given that this is an OSS repo, I think we can't have this in the readme as it will confuse people who don't have this `module` system.

Reply:
I just pushed this change to your branch!